test-network/network.sh

```diff
@@ -236,27 +230,7 @@ function createOrgs() {
 # You can ignore the logs regarding intermediate certs, we are not using them in
 # this crypto implementation.
 
-# Generate orderer system channel genesis block.
-function createConsortium() {
-  which configtxgen
-  if [ "$?" -ne 0 ]; then
-    fatalln "configtxgen tool not found."
-  fi
```

Comment:
This check for configtxgen existence needs to be moved into the remaining invocation of configtxgen, in createChannel.sh createChannelGenesisBlock() function.  This gives users a good message if configtxgen is not found.

Additionally, the comments in network.sh lines 271-274 are now outdated.  configtxgen is not called "once more". It will only be called that one time. And configtx.yaml is not mounted in the CLI container anymore.